variables.tf

```diff
@@ -13,3 +13,35 @@ variable "aws_profile" {
   type        = string
   description = "The profile that was used to deploy this module. If the default profile / default credentials are used, seet this value to null."
 }
+
+variable "api_token" {
+  type        = string
+  description = "This is your Bridgecrew platform Api token Set as and Environment variable TF_VAR_api_token"
+}
+
+variable "template_version" {
+  type        = string
+  default     = "1.5"
+  description = "Version of this platform template"
+}
+
+variable "common_tags" {
+  type        = map(any)
+  description = "Implements the common tags scheme"
+  default = {
+    "module"      = "terraform-aws-bridgecrew-read-only"
+    "integration" = "bridgecrew-aws-readonly"
+  }
+}
+
+
+variable "topic_name" {
```

Comment:
Same comments here as on the TF remediation module - adding these as vars, even with defaults, will be confusing for the user.

Reply:
If you modifiy the local in the module when youre testing youll run the risk of breaking the thing? Thats what i trying to avoid really

Reply:
I'm down to leave these with variables as defaults and see what happens. If I am wrong and customers find it easy to use, then you are right that it will make it easier to work with in the future.